website/docs/language/resources/syntax.mdx

```diff
@@ -117,6 +117,37 @@ For more information about how OpenTofu manages resources when applying a
 configuration, see
 [Resource Behavior](/docs/language/resources/behavior).
 
+## Removing Resources
+
+By default, deleting a resource block from your configuration will result in a plan to destroy it, effectively removing
+the infrastructure object described by the resource.
+
+However, there are instances when you want to remove a resource from your configuration without destroying the
+corresponding infrastructure object. In such cases, you can remove it from the [OpenTofu state](/docs/language/state)
+while allowing it to persist in the remote system.
+
+To achieve this, follow these steps:
+1. Delete the resource from your configuration.
+2. Add a removed block instead, specifying the resource address you want to "forget" in the `from` attribute. Note that
+the address cannot include instance keys (for example, "aws_instance.web[0]").
```

Comment:
The note block could be moved into an [admonition](https://docusaurus.io/docs/markdown-features/admonitions).

Reply:
✅